index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+  </head>
+  <body>
+    <header>
+      <nav id="navbar">
+        <div class="container">
+          <a href="#" class="page_logo">My Logo</a>
+          <div class="navbar-nav">
+            <button type="button" class="nav-toggle-btn">
+              <img
+                src="./assets/images/icons/Union@2x.png"
+                width="3rem"
+                alt="menu-bar"
+              />
```

Comment:
- [ ] Please The `<header>` element represents a container for introductory content or a set of navigational links it would be better to contain just the toolbar section inside the `header` tag.